test/it/inlinehook-crud.js

```diff
@@ -83,7 +84,7 @@ describe('Inline Hook Crud API', () => {
     });
 
     it('should update name for created inlineHook', async () => {
-      inlineHook.name = 'Mock OIDC/OAuth token inline hook updated';
+      inlineHook.name = `node-sdk: Mock inline hook updated ${faker.random.word()}`.substring(0, 49);
```

Comment:
Is there any need to prefix this to show what/who to blame if it gets left lying around?

Reply:
Actually, now that we have separate orgs for each of the SDKs, this might not be needed.
But in case, we run tests against an existing org against which other SDK tests have run, it'll be easier to identify those who don't clean up after themselves.